test/ci/ci_tests.cfg

```diff
@@ -795,14 +928,63 @@ args=%(STDARGS_ICARUSCODE)s --input-file %(INPUT_STREAM)s --reference-files %(RE
 parse_art_output=True
 output1=%(TFILENAME)s
 
+[test nucosmics_reco0_seq_test_icaruscode]
+STAGE_NAME=reco0
+INPUT_STAGE_NAME=detsim
+NEVENTS=%(NEVENTS_SEQ_NUCOSMICS_ICARUSCODE)s
+
+# calibrated with [e19] on icarusbuild02.fnal.gov
```

Comment:
This message needs to be updated... I suggest adding the date of calibration too (see some other messages).